src/Wellcome.Dds/Wellcome.Dds.Server/Controllers/ThumbController.cs

```diff
@@ -60,38 +66,85 @@ public async Task<IActionResult> Index(string id)
                 return RedirectPermanent(iiifThumbs[0].Id);
             }
             
-            // TODO - PDF Cover thumbs - use Cantaloupe!
             if (manifestation.AssetType == "image/jp2")
             {
                 return NotFound($"No thumbnail for {id}");
             }
-            
+
             IMetsResource resource = await metsRepository.GetAsync(id);
+            if (resource is IManifestation {FirstInternetType: "application/pdf"})
```

Comment:
@tomcrane - is this logic correct to discern whether we're dealing with a PDF?

Reply:
I think this can be `if (manifestation.AssetType == "application/pdf")` in keeping with the one above.